src/tlo/methods/healthsystem.py

```diff
@@ -1687,7 +1721,13 @@ def log_current_capabilities_and_usage(self):
             summary_by_officer['Minutes_Used'] / summary_by_officer['Total_Minutes_Per_Day']
         ).replace([np.inf, -np.inf, np.nan], 0.0)
         summary_by_officer.index.names = ['Officer_Type', 'Facility_Level']
-
+        
+        # Store information on fraction of time used, broken down by officer type and level
+        summary_by_officer['OfficerType_FacilityLevel'] = summary_by_officer.index.get_level_values('Officer_Type') + '_' + summary_by_officer.index.get_level_values('Facility_Level')
+        
+        for index, row in summary_by_officer.iterrows():
+            self._summary_counter._frac_time_used_by_officer_type_and_level[row['OfficerType_FacilityLevel']] += float(row['Fraction_Time_Used'])
+ 
```

Comment:
Accessing an attribute of another object prefixed with an underscore `_` should generally be avoided as the convention is that these are for internal use only within the object (so accessing `self._summary_counter` here is fine as `_summary_counter` is an internal attribute of the module, but `_summary_counter._frac_time_used_by_officer_type_and_level` should only be accessed by the `HealthSystemSummaryCounter` class). 

One option would be to just mark `_frac_time_used_by_officer_type_and_level` as for external use by removing `_` prefix, but I would say it would be better here to extend the existing `HealthSystemSummaryCounter.record_hs_status` method to deal with also updating the fractions of time by officer type and level counter as something like
```Python
    def record_hs_status(
        self,
        fraction_time_used_across_all_facilities: float,
        fraction_time_used_by_officer_type_and_level: Dict[Tuple[str, int], float],  
    ) -> None:
        """Record a current status metric of the HealthSystem."""
        # The fraction of all healthcare worker time that is used:
        self._frac_time_used_overall.append(fraction_time_used_across_all_facilities)
        for officer_type_facility_level, fraction_time in fraction_time_used_by_officer_type_and_level.items():
            self._sum_frac_time_used_by_officer_type_and_level[officer_type_facility_level] += fraction_time
```
and then calling here by replacing the current line 1741 with
```
    self._summary_counter.record_hs_status(
        fraction_time_used_across_all_facilities=fraction_time_used_overall,
        fraction_time_used_by_officer_type_and_level=summary_by_officer["Fraction_Time_Used"].to_dict()
    )
```

I would also say an attribute name `_sum_frac_time_used_by_officer_type_and_level` would be more explicit that this counter is recording sum of fractions of time hence the above also suggests changing this name.

Reply:
Note this also changes the `_sum_frac_time_used_by_officer_type_and_level ` counter to be keyed by a tuple of officer type and level rather than a concatenated string.